backend/src/main/resources/application.yml

```diff
@@ -0,0 +1,72 @@
+server:
+  port: 8080
+
+spring:
+  application:
+    name: FSA Forest Client
+  main:
+    lazy-initialization: false
+    banner-mode: "off"
+    allow-bean-definition-overriding: true
+  r2dbc:
+    url: r2dbc:postgresql://${ca.bc.gov.postgres.host}/${ca.bc.gov.postgres.database}
+    username: ${ca.bc.gov.postgres.username}
+    password: ${ca.bc.gov.postgres.password}
+    pool:
+      enabled: true
+      initialSize: 1
+      minIdle: 1
+      maxSize: 3
+      maxLifeTime: 60000
+      maxIdleTime: 45000
+      maxCreateConnectionTime: 90000
+      poolName: FsaClientConPool
+  flyway:
+    locations: classpath:db/migration
+    enabled: true
+    validate-on-migrate: true
+    user: ${ca.bc.gov.postgres.username}
+    password: ${ca.bc.gov.postgres.password}
+    url: jdbc:postgresql://${ca.bc.gov.postgres.host}/${ca.bc.gov.postgres.database}
+
+management:
+  endpoint:
+    health:
+      show-details: always
+  endpoints:
+    web:
+      base-path: /
+      exposure:
+        include: "*"
+  httpexchanges:
+    recording:
+      include: principal,request-headers,response-headers,cookie-headers,time-taken,authorization-header,remote-address,session-id
+
+springdoc:
+  webjars:
+    prefix: /v3
+  api-docs:
+    groups:
+      enabled: true
+  show-actuator: false
+
+frontend:
+  url: ${FRONTEND_URL:*}
+
+
+ca:
+  bc:
+    gov:
+      ches:
+        uri: http://127.0.0.1:10010/chess/uri
+        tokenUrl: http://127.0.0.1:10010/token/uri
+        clientId: clientId
+        clientSecret: secret
+        scope: scope
+      orgbook:
+        uri: https://orgbook.gov.bc.ca/api
+      postgres:
```

Comment:
can we move this section to ca.bc.gov.nrs

Reply:
Done